scripts/repository.lic

```diff
@@ -467,6 +468,21 @@ add_comments_to_list = proc { |list|
   end
 }
 
+add_links_to_list = proc { |list, action| # Make file name clickable t
```

Comment:
Used a proc to keep same style. One minor issue this caused is not being able to set a default for action, just means everywhere this function is called acton needs to be provided (required or not). If not set as first parameter is an array it is automatically deconstructed (unless a way I missed to avoid this?)